content/guides/references/client_certificates.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Certificates
+---
+
+It is possible in Cypress to configure CA and client certificates to use within tests on a per URL basis.
+
+<Alert type="info">
+
+<strong class="alert-header">How to specify client certificate info</strong>
+
+This document is only a reference to offer guidance on how to specify certificate file paths for given test URLs. For information on the content and purpose of such files, please search elsewhere as this is not within the scope of Cypress documentation.
+
+</Alert>
+
+## Structure within JSON config file
+
+To configure CA / client certificates within your cypress.env.json, you should add a section at the top level something like:
+
+```
+  "clientCertificates": [
+    {
+      "url": "https://a.host.com",
+      "ca": [
+          "certs/ca.pem"
+      ],
+      "certs": [
+        {
+          "cert": "certs/cert.pem",
+          "key": "certs/private.key",
+          "passphrase": "certs/pem-passphrase.txt"
+        }
+      ]
+    },
+    {
+      "url": "https://b.host.com/a_base_route/**",
+      "ca": [],
+      "certs": [
+        {
+          "pfx": "/home/tester/certs/cert.pfx",
+          "passphrase": "/home/tester/certs/pfx-passphrase.txt"
+        }
+      ]
+    },
+    {
+      "url": "https://a.host.*.com/",
+      "ca": [],
+      "certs": [
+        {
+          "pfx": "certs/cert.pfx",
+          "passphrase": "certs/pfx-passphrase.txt"
+        }
+      ]
+    }
+  ]
+}
+```
```

Comment:
Where is the schema for this?

Reply:
This is in the main cypress.schema.json file, or rather, it will be when https://github.com/cypress-io/cypress/pull/15179/ is merged :)